extensions/toggl-track/README.md

```diff
@@ -0,0 +1,41 @@
+# Raycast Toggl Track Extension
+
+## Getting started
+
+To use the extension you need to submit your API Token. This can be found on the bottom of your profile in toggle track: https://track.toggl.com/profile
+
+## Features
+
+### Show the running time entry and stop it
+
+<img width="600" alt="Screenshot 2022-01-23 at 01 57 00" src="https://user-images.githubusercontent.com/22563154/150660444-88b9ed81-d5d5-4aed-9bf7-41d8972dce3c.png">
+
+The running time entry is always shown on the top of the window with its duration, description and project. It is easily stopped by clicking on it or using the enter command.
+
+### Start a new time entry
+
+<img width="600" alt="Screenshot 2022-01-23 at 01 55 14" src="https://user-images.githubusercontent.com/22563154/150660414-b551e659-64c3-4033-a289-ff8fa895d0b4.png">
+
+* Add project
+* Add description
+* Add tags
+
+### Resume a recent time entry
+
+<img width="600" alt="Screenshot 2022-01-23 at 01 58 44" src="https://user-images.githubusercontent.com/22563154/150660477-b3d9fef0-a088-4302-8e99-d89c891f9fdb.png">
+
+The extension lists out recent time entries and allows you to resume them.
+
+### Multiple workspaces and clients
+
+The extension supports multiple workspaces and clients out of the box, and will group projects by both. This resembles how projects are grouped when selecting them on the official web app (https://track.toggl.com).
```

Comment:
```suggestion
The extension supports multiple workspaces and clients out of the box, and will group projects by both. This resembles how projects groups when selecting them on the official web app (https://track.toggl.com).
```

Reply:
I don't think this is correct spelling? Fixed the rest of your suggestions :)